lib/config/config-file.js

```diff
@@ -371,6 +350,11 @@ function resolve(filePath) {
             filePath = "eslint-config-" + filePath;
         }
 
+        // some additional information processing is necessary
+        filePath = resolveModule.sync(filePath, {
+            basedir: relativeTo || process.cwd()
```

Comment:
Going to leave this for now. It will be an edge case for now, but I'll come back around and refactor later to do this correctly.